src/main/java/com/pokegoapi/auth/GoogleLogin.java

```diff
@@ -16,9 +16,11 @@
 package com.pokegoapi.auth;
 
 import POGOProtos.Networking.Envelopes.RequestEnvelopeOuterClass.RequestEnvelope.AuthInfo;
+
```

Comment:
Those new lines are there to segregate different import categories. XML_LEGACY_SETTINGS_IMPORTED check-style rule allows it and I believe the default checkstyle rule requires it.